runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -383,6 +445,60 @@ else if (activeTab != null
 			tagManager.removeTag(itemId, activeTab.getTag());
 			doSearch(InputType.SEARCH, TAG_SEARCH + activeTab.getTag());
 		}
+		else if (activeTab != null
```

Comment:
you should probably merge these 2 ifs, and do the activeTab difference inside (also you could reduce amount of code like this, because you can just build the list, and use .addTags() in both cases

Reply:
One opens the chatbox, the other doesn't